kitsune/sumo/static/sumo/js/device-migration-wizard.js

```diff
@@ -2,22 +2,117 @@ import trackEvent from "sumo/js/analytics";
 import SwitchingDevicesWizardManager from "sumo/js/switching-devices-wizard-manager";
 import { BaseFormStep } from "sumo/js/form-wizard";
 
+import keyImageURL from "sumo/img/key.svg";
+
 class SignInStep extends BaseFormStep {
   get template() {
     return `
       <template>
-        <div>
-          <p>This is where the sign in/sign up form will go.</p>
-          <p id="email"></p>
+        <div id="sign-in-step-root">
+          <p>
+            ${gettext("Body provide short value of the account other than the bookmarks/password/history line, what else could they get out of it that would be good to know here?")}
+          </p>
+
+          <p class="for-sign-up">
+            <strong>${gettext("Create an account")}</strong>
+          </p>
+          <p class="for-sign-in">
+            <strong>${gettext("Sign in to your account")}</strong>
+          </p>
+
+          <form method="get">
+            <input name="service" value="sync" type="hidden"/>
+            <input name="action" value="email" type="hidden"/>
+            <input name="context" value="" type="hidden"/>
+            <input name="flow_id" value="" type="hidden"/>
+            <input name="flow_begin_time" value="" type="hidden"/>
+            <input name="utm_source" value="" type="hidden"/>
+            <input name="utm_medium" value="" type="hidden"/>
+            <input name="entrypoint" value="" type="hidden"/>
+            <input name="redirect_immediately" value="true" type="hidden"/>
+            <input name="redirect_to" value="" type="hidden"/>
+
+            <label for="email">${gettext("Email")}</label>
+            <input id="email" name="email" type="email" required="true" placeholder="${gettext("Enter your email")}"/>
+
+            <input class="for-sign-up" type="submit" value="${gettext("Sign up")}"/>
+            <input class="for-sign-in" type="submit" value="${gettext("Sign in")}"/>
+          </form>
+
+          <p class="for-sign-up">
+            ${gettext("Already have an account?")} <a class='alternative-link' href='#' data-event-category="device-migration-wizard" data-event-action="click" data-event-label="sign-in-link">${gettext("Sign in")}</a>
+          </p>
+          <p class="for-sign-in">
+            ${gettext("Don’t have an account?")}<a class='alternative-link' href='#' data-event-category="device-migration-wizard" data-event-action="click" data-event-label="signup-link">${gettext("Sign up")}</a>
```

Comment:
Oops, I just noticed that I mistakenly removed a space here!
```suggestion
            ${gettext("Don’t have an account?")} <a class='alternative-link' href='#' data-event-category="device-migration-wizard" data-event-action="click" data-event-label="signup-link">${gettext("Sign up")}</a>
```